Cognite.Simulator.Utils/ConnectorBase.cs

```diff
@@ -177,8 +199,22 @@ protected async Task EnsureSimulatorIntegrationsSequencesExists(CancellationToke
         /// </summary>
         protected async Task UpdateIntegrationRows(
             bool init,
-            CancellationToken token)
+            CancellationToken token,
+            bool licenseCheck = false,
+            string licenseCheckStatus = "")
         {
+            if (licenseCheck is true)
```

Comment:
assigning values inside this if statement should be moved into `public async Task LicenseCheck(CancellationToken token)`.
init value for the status could be "Not checked yet", after one hour it'll be overwritten.

this makes this method more lightweight, only reading the current LastLicenseCheckTimestamp/LastLicenseCheckResult values instead.

Reply:
Fixed but had to keep this in the function: 
`if (licenseCheck && init)
            {
                LastLicenseCheckResult = "Not checked yet";
            }`